ansible/roles/ocp4-workload-security-compliance-lab/defaults/main.yml

```diff
@@ -1,110 +1,149 @@
 ---
-# PROVIDED BY THE INFRA
-# Uncomment and set a value to take effect. Else, will use the defaults
+silent: False
 admin_project: ocp-workshop
-user_count_start: 1
-num_users: 5 
-#user_format: user%02d
+guid: raleigh-fb78
+domain: "{{ guid }}.openshiftworkshop.com"
+bastion: "bastion.{{ domain }}"
+tmp_dir: "/tmp/{{ guid }}"
+
+###### User defaults
+user_count_start: 0
+user_count: 3
```

Comment:
need to figure out a way to get this value from the `ocp4_workload_authentication` role